src/env-inl.h

```diff
@@ -291,6 +292,8 @@ inline void Environment::AssignToContext(v8::Local<v8::Context> context,
   // Used to retrieve bindings
   context->SetAlignedPointerInEmbedderData(
       ContextEmbedderIndex::kBindingListIndex, &(this->bindings_));
+  context->SetAlignedPointerInEmbedderData(
+      ContextEmbedderIndex::kPolicyIndex, this->policy_.get());
```

Comment:
Can you leave a TODO comment here that indicates that it would be nice to group all the per-Context data in a C++ struct? I feel like that would make lifetime management for these a lot easier

Reply:
Yeah, good idea, it's a bit of a mess in here right now.